src/django/api/tests.py

```diff
@@ -350,7 +350,7 @@ def test_get_with_authentication_token_returns_items(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.json()), 1)
 
-    def test_upload_by_user_with_no_contributor_returns_400(self):
+    def test_upload_by_user_with_no_contributor_returns_402(self):
```

Comment:
We now send a 402 in response to requests for access by users with no contributor.